layouts/add-attribute.html

```diff
@@ -9,11 +9,11 @@
                 <div class="col-md-9">
                     <div class="input-group">
                         <input type="text" class="form-control" ng-model="mc.label" ng-model-options="{ debounce: 250 }" placeholder="Search" uib-typeahead="result as result.label for result in mc.onSearch($viewValue)" typeahead-min-length="3" typeahead-wait-ms="50" typeahead-loading="loadingResultsLabel" typeahead-no-results="noResultsLabel">
-                        <span class="input-group-addon"><i class="fa fa-fw fa-search"></i></span>
+                        <span class="input-group-addon"><i class="material-icons">search</i></span>
                     </div>
-                    <i ng-show="loadingResultsLabel" class="fa fa-fw fa-refresh"></i>
+                    <i ng-show="loadingResultsLabel" class="material-icons">refresh</i>
                     <div ng-show="noResultsLabel && mc.label.length >= 3">
-                        <i class="fa fa-fw fa-times"></i> No Results Found
+                        <i class="material-icons md-18">block</i> No Results Found
```

Comment:
I prefer the clear icon. Additionally that's the icon they use in their examples. Any preference?